react/features/feedback/components/FeedbackDialog.web.js

```diff
@@ -176,6 +187,12 @@ class FeedbackDialog extends Component<Props, State> {
         if (typeof APP !== 'undefined') {
             APP.API.notifyFeedbackPromptDisplayed();
         }
+        if (this.props.timeout && this.props.timeout > 0) {
+            this._timeout = setTimeout(() => {
+                this._onCancel();
+                this.props.dispatch(hideDialog());
+            }, this.props.timeout);
+        }
```

Comment:
Not sure we want to timeout in 60 if someone is actively typing, for example. Should this be an idle timeout?

Reply:
yeah, good point

Reply:
that also means it's not a good fix for spot

Reply:
or we can fix it on the spot side.

Reply:
yes, I guess if we want to track the input it needs to be fixed on the spot side